### ISSUE TITLE
https://beego.vip/ - Not working 

### ISSUE THREAD

**Issue by msiripurapu05** [beego/beego#5151]
I am working on a project implemented using Beego 1.12. We are now analyzing the possibility and benefits of migrating to Beego 2.07. Unfortunately the official site it seems down <URL>.
Can you please share the documentation links for the following,
* Migration guide from 1.x to 2.x
*  New features / advantages of 2x.

I would like to know how long you’re planning to provide support for Beego 1.x.

**Comment by dominhtri1995** [comment]
Agree. It has been so frustrating when the official docs page is down.

**Comment by goldsky** [comment]
does this help?
https://web.archive.org/web/20220524230410/https://beego.vip/docs/intro/

**Comment by Roccoriu** [comment]
Unfortunately using the webarchive is not possible for our team at work. Is it possible to have this fixed sometime soon? It's not a good look if the official website points to a gambling page. 

![Screenshot from 2023-04-29 09-03-54](https://user-images.githubusercontent.com/25778176/235288906-a13fdf4b-110b-4374-8861-15dcd1ca8c0b.png)

![Screenshot from 2023-04-29 09-04-05](https://user-images.githubusercontent.com/25778176/235288928-a1a5f98a-9f04-42a4-b846-cb05e8868802.png)

**Comment by mobilemindtec** [comment]
Hi,


maybe https://beego.gocn.vip/beego/en-US/developing/

**Comment by Roccoriu** [comment]
@mobilemindtec unfortunately this is blocked for me at work. I'm sure I am not the only one to face the issue that .vip domains are blocked by the company policy. Is there a reason for why you use a .vip domain?

**Comment by flycash** [comment]
we do not have a valid HTTPS cert 😭

**Comment by flycash** [comment]
Or you can access the source https://github.com/beego/beego-doc directly
